jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/Resource.java

```diff
@@ -852,6 +719,7 @@ public Resource resolve(String subUriPath) throws IOException
         return newResource(resolvedUri);
     }
 
+    // TODO: move to URIUtil
     private static URI uriResolve(URI uri, String subUriPath) throws IOException
```

Comment:
Move to `URIUtil` maybe? :-)

Reply:
I'm going to get rid of this, as it doesn't make much sense anymore.